Reintroduce query plan reuse during warmup (#5255)
apollographql/router: docs/source/configuration/in-memory-caching.mdx

    experimental_reuse_query_plans: true
```

> To check if the Router deployment would benefit from this option, check the counter metric named `apollo.router.query.planning.warmup.reused` that indicates how many query plans were reused in warmup if its `query_plan_reuse_active` is true, or how many query plans would have been reused if the option was active, when `query_plan_reuse_active` is false.
> 

Reviewer: ```suggestion
To decide if a Router deployment would benefit from this option, check the counter metric named `apollo.router.query.planning.warmup.reused`. This metric indicates how many query plans were reused, or would have been reused, depending on  whether `query_plan_reuse_active` is true or false.
```

Author: that makes it less clear what `query_plan_reuse_active` does. Is there another way to rephrase it?
